arm/Microsoft.EventHub/namespaces/deploy.bicep

```diff
@@ -127,7 +127,7 @@ param diagnosticMetricsToEnable array = [
 ]
 
 var maxNameLength = 50
-var uniqueEventHubNamespaceUntrim = uniqueString('EventHub Namespace${baseTime}')
+var uniqueEventHubNamespaceUntrim = 'evhns-${uniqueString(resourceGroup().id)}'
```

Comment:
@eriqua is that what you were going for in the issue? 

The obivous advantage is that the name isn't new every time the deployment runs. However, does it also ensure that no numbers are part of the `uniqueString` ?

Reply:
Numbers are not accepted in the first position but can be used after so it should be fine to have the uniqueString as a suffix.
A couple of comments:

- If the `baseTime` parameter is not used anymore we should remove it and regenerate the readme file
- The uniqueString function returns a string of 13 chars so the substring function in the next line should never be used since length(uniqueEventHubNamespaceUntrim) will never be greater than 50. Hence I'd also suggest to clean up that code